src/app/content/reducer.ts

```diff
@@ -20,6 +21,7 @@ import { State } from './types';
 
 export const initialState = {
   buyPrint: null,
+  experiments: initialExperimentsState,
```

Comment:
i think i want this outside of `content` and maybe we should name it `featureFlags` instead of `experiments`